### ISSUE TITLE
Incomplete / incorrect SDK types of YYGMLFuncs

### ISSUE THREAD

**Issue by Archie-osu** [AurieFramework/YYToolkit#45]
Correct type is:
```cpp
struct YYGMLFuncs
{
	const char* pName;
	union
	{
		PFUNC_YYGMLScript pScriptFunc;
		PFUNC_YYGML pFunc;
	};
	YYVAR* pFuncVar;
};
```

The ``pScriptFunc`` type is what is used in YYC script calls (see Will You Snail? for an example game).

**Comment by Archie-osu** [comment]
Forgot to add, but type of PFUNC_YYGMLScript is
```cpp
using PFUNC_YYGMLScript = YYRValue*(*)(CInstance* _self, CInstance* _other, YYRValue* _return_value, int _argc, YYRValue** args);
```

**Comment by Archie-osu** [comment]
Fixed in 5289608c20d91f656d5c7d6b99b62176d4d4a015.
